dubbo-common/src/main/java/org/apache/dubbo/common/utils/NetUtils.java

```diff
@@ -231,21 +216,31 @@ public static InetAddress getLocalAddress() {
         return localAddress;
     }
 
+    private static Optional<InetAddress> toValidAddress(InetAddress address) {
+        if (address instanceof Inet6Address) {
+            Inet6Address v6Address = (Inet6Address) address;
+            if (isValidV6Address(v6Address)) {
+                return Optional.ofNullable(normalizeV6Address(v6Address));
+            }
+        }
+        if (isValidAddress(address)) {

```

Comment:
Better to refactor the method name isValidAddress to **isValidV4Address**. What do you say?